webpack.dev.config.js

```diff
@@ -60,6 +60,8 @@ module.exports = {
             loader: 'css-loader',
             options: {
               sourceMap: true,
+              importLoaders: 2,
+              localIdentName: '[name]--[local]--[hash:base64:8]',
```

Comment:
Doesn't this break all previously declared global css?

Reply:
@andre-meyer Right now it doesn't, it will break after we set `modules: true`, this is just some kind of preparation for names 😃